graphql/data/schema.js

```diff
@@ -765,10 +766,16 @@ const userType = new GraphQLObjectType({
       resolve: (user, _, context) =>
         getShouldShowSfacExtensionPrompt(context.user, user),
     },
+    sfacActivityState: {
+      type: new GraphQLNonNull(GraphQLString),
+      description: 'what mode in which to show SFAC searches UI',
+      resolve: (user, _, context) => getSfacActivityState(context.user, user),
+    },
     yahooPaidSearchRewardOptIn: {
       type: new GraphQLNonNull(GraphQLBoolean),
-      description:
-        'whether or not the user has opted into searching for extra impact',
+      description: 'status ',
+      resolve: (user, _, context) =>
+        getShouldShowSfacExtensionPrompt(context.user, user),
```

Comment:
Are these changes to yahooPaidSearchRewardOptIn intentional? If so, could you help me understand what they're doing?

Reply:
Err.... no idea! My bad!